l5kit/l5kit/simulation/utils.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+
+from l5kit.data import ChunkedDataset, get_agents_slice_from_frames
+
+
+def insert_agent(agent: np.ndarray, frame_idx: int, dataset: ChunkedDataset):
```

Comment:
Given that you're changing the `dataset (ChunkedDataset)`, would it make more sense to move this to be a method on the `ChunkedDataset` or to a wrapper around it ? Just to move it outside from the `utils` module.

Reply:
it might be, but I'm worried it will accumulate a lot of methods that way...Also, this only works if the dataset is in memory, so it won't work for a default `ChunkedDataset`. This is probably the wrapper we should have